packages/shared/styles/variables/_colors.scss

```diff
@@ -26,6 +27,7 @@
   --c-body: var(--_c-light-secondary);
   --c-text: var(--_c-dark-primary);
   --c-text-muted: var(--_c-gray-primary);
+  --c-text-disabled: var(--_c-gray-accent);
```

Comment:
@aniamusial can you give me feedback when we should use muted and when disabled? in not the same?

Reply:
In most cases on designs muted is on on :pressed/ :hovered - :disabled is used on well, disabled - see SfButton/SfCircleIcon for reference

Reply:
awesome, thanks :) for me it sounds fine